mp4parse_capi/tests/test_sample_table.rs

```diff
@@ -95,3 +95,59 @@ fn parse_sample_table() {
         mp4parse_free(parser);
     }
 }
+
+#[test]
+fn parse_sample_table_with_elst() {
+    let mut file = std::fs::File::open("tests/short-cenc.mp4").expect("Unknown file");
+    let io = mp4parse_io {
+        read: buf_read,
+        userdata: &mut file as *mut _ as *mut std::os::raw::c_void
+    };
+
+    unsafe {
+        let parser = mp4parse_new(&io);
+
+        let mut rv = mp4parse_read(parser);
+        assert_eq!(rv, mp4parse_error::MP4PARSE_OK);
+
+        let mut counts: u32 = 0;
+        rv = mp4parse_get_track_count(parser, &mut counts);
+        assert_eq!(rv, mp4parse_error::MP4PARSE_OK);
+        assert_eq!(counts, 2);
+
+        let mut track_info = mp4parse_track_info {
+            track_type: mp4parse_track_type::MP4PARSE_TRACK_TYPE_AUDIO,
+            codec: mp4parse_codec::MP4PARSE_CODEC_UNKNOWN,
+            track_id: 0,
+            duration: 0,
+            media_time: 0,
+        };
+        rv = mp4parse_get_track_info(parser, 1, &mut track_info);
+        assert_eq!(rv, mp4parse_error::MP4PARSE_OK);
+        assert_eq!(track_info.track_type, mp4parse_track_type::MP4PARSE_TRACK_TYPE_AUDIO);
+        assert_eq!(track_info.codec, mp4parse_codec::MP4PARSE_CODEC_AAC);
+
+        // Check audio smaple table
+        let mut is_fragmented_file: u8 = 0;
```

Comment:
The test would be more robust if you initialize to an impossible value, like `2u8` to verify the method is setting the expected value, which is also zero in this case.